addons/website/views/new_page_template_templates.xml

```diff
@@ -538,22 +538,22 @@ overridden by modules), because:
 
 <template id="new_page_template_about_full_s_numbers" inherit_id="website.new_page_template_about_s_numbers" primary="True">
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="o_cc3 o_colored_level" remove="o_cc2" separator=" "/>
+        <attribute name="class" add="o_cc3 o_colored_level" separator=" "/>
```

Comment:
If the base template does not have `o_cc` anymore - it should be added in variants that add any `o_ccN` => new page templates, configurator templates (if any), and in the themes customization.